app/src/main/res/layout/section_title.xml

```diff
@@ -33,7 +33,7 @@
       android:layout_marginEnd="@dimen/section_title_text_view_layout_margin_end"
       android:fontFamily="sans-serif-medium"
       android:text="@{viewModel.sectionTitleText}"
-      android:textColor="@color/component_color_recently_played_activity_section_title_color"
+      android:textColor="@color/component_color_shared_content_item_text_view_color"
```

Comment:
Dont use `component_color_shared_content_item_text_view_color ` instead use `component_color_shared_primary_text_color`

Reply:
Okay I see everywhere I used this text, I replace all with `component_color_shared_primary_text_color `